students/adam_wulw/lesson_03_functions/calculator.py

```diff
@@ -0,0 +1,97 @@
+def disp_help():
+    """Display function options."""
+    print("Welcome to well organized calculator:")
+    print("a - add")
+    print("s - subtract")
+    print("m - multiply")
+    print("d - divide")
+    print("p - power")
+    print("h,? - help")
+    print("q - QUIT")
+
+
+def math_add():
+    """Reads two int value and print sum of them."""
+    print("ADDING")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 + add_var_2)
+
+
+def math_sub():
+    """Reads two int value and print subtract of them."""
+    print("SUBTRACT")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 - add_var_2)
+
+
+def math_multi():
+    """Reads two int value and print quotient of them."""
+    print("MULTIPLY")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 * add_var_2)
+
+
+def math_div():
+    """Reads two int value and print their division."""
+    print("DIVIDE")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 / add_var_2)
+
+
+def math_pow():
+    """Reads two int value and print their power."""
+    print("POWER")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 ** add_var_2)
+
+
+disp_help()
+while True:
+    print("Enter option:")
+
+    option = raw_input()
+
+    if option == "a":
```

Comment:
You can build sth like:
options = {
    'a':  math_add,
    ...
}

and later options[option]()